src/main/java/org/advent/dayten/Pipe.java

```diff
@@ -0,0 +1,22 @@
+package org.advent.dayten;
+
+public enum Pipe {
+
+//    L_PIPE,
+//    F_PIPE,
+//    J_PIPE,
+//    SEVEN_PIPE,
+//    HOR_PIPE,
+//    VERT_PIPE;
+//
+//    public CoordinateDirection getDirection(String typeOfPipe) {
+//        if (typeOfPipe.equals(L_PIPE)) {
+//            return "hey"
+//        }
+//
+//    }
+
```

Comment:
this isnt a bad idea, a pipe should connect 2 directions and know how to decided which of the directions is incomming and which one is outgoing